src/plot_api/to_image.js

```diff
@@ -0,0 +1,106 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+/*eslint dot-notation: [2, {"allowPattern": "^catch$"}]*/
```

Comment:
Let's remove the `{"allowKeywords": false}` option in the [`eslintrc`](https://github.com/plotly/plotly.js/blob/master/.eslintrc#L41) file and remove this line.